index.html

```diff
@@ -1492,13 +1490,44 @@ <h2 id="state-transitions" class="informative">
         </dd>
       </dl>
     </section>
+    <section data-dfn-for="PaymentComplete" data-link-for="PaymentComplete">
+      <h2><dfn>PaymentComplete</dfn> enum</h2>
+      <pre class="idl">
+        enum PaymentComplete {
+          "fail",
+          "success",
+          "unknown",
+        };
+      </pre>
+      <dl>
+        <dt>
+          "<dfn>fail</dfn>"
+        </dt>
+        <dd>
+          Indicates that processing of the payment failed. The <a>user
+          agent</a> MAY display UI indicating failure.
```

Comment:
It's a bit weird that these normative processing model requirements are given in the definition of the enum, instead of in the algorithm that processes it, but that's a separate issue.